runtime/src/accounts_db.rs

```diff
@@ -220,6 +220,16 @@ pub struct ErrorCounters {
     pub invalid_writable_account: usize,
 }
 
+#[derive(Debug, Default, Clone, Copy)]
+pub struct GenerateIndexResult {}
+
+#[derive(Debug, Default, Clone, Copy)]
+struct GenerateIndexForSlotResult {
```

Comment:
Can we come up with a name for these that are descriptive of the data, rather than the methods that spit them out?
Also, things that are called `Result` are usually expected to be or contain a Result type...

Reply:
Totally agree on the `Result` comment. And yes, happy to use a better name, but nothing comes to my mind 😞 Do you have suggestions?

Reply:
Heh, I see what you mean. Maybe something like `SlotIndexAccountStats`?
I guess even `GenerateIndexForSlotStats` would be an improvement to my mind.

Reply:
What about `Info`[1] instead of `Result`? Since the collection of values being returned doesn't have an obvious name, I feel like getting the function name is the type name is beneficial...

Other options:
- Resultant
- Outcome
- SideEffect
- Consequence
- Conclusion

[1] I asked on the Rust Community Discord and a person with similar use cases suggested `Info`. See [here](https://discord.com/channels/273534239310479360/274215136414400513/918586639670996993) for more info.

Reply:
Info seems as good as Stats to me, so happy to go with that.
However, the issue with using that function name -- which I alluded to in my edit to my previous comment -- is that there's no way to disambiguate between "the info which comes from generating the index for a slot" and "the index which is generated for a SlotInfo". So it ends up being not self-documenting.

`SlotIndexGenerationInfo`?

Reply:
Done!

Reply:
PS. Would love to see the info in Rust Community Discord, but discord links are trash and I can't find it, even though I think I am in that server 😞 (Rust Programming Language Community Server, right?)

Reply:
> Done!

Sweet, I like it! Thanks for indulging me

Reply:
Yeah... I always have trouble with Discord links. I assumed it was just me, but I guess the links are bad for everyone; boo!

And yes, that's the right server. It was in #rust-questions. I'm @brooks there as well. Was 1:35pm (Central time) for the link.

Reply:
Perfect, I found it, thanks!